### PR TITLE
Sync OTA provider device type with specs

### PR DESCRIPTION
**Pull request by jadhavrohit924**
#### Problem
What is being fixed?  Examples:
* OTA provider device type was not matching specs

#### Change overview
Sync OTA provider device type with specs

#### Testing
How was this tested? (at least one bullet point required)
* Verified OTA provider device type syncs with specs

### CODE REVIEW

File: src/app/zap-templates/zcl/data-model/chip/matter-devices.xml [hunk 1]
```diff
@@ -144,22 +144,12 @@ limitations under the License.
...
                 <requireAttribute>DEVICE_TYPE_LIST</requireAttribute>
                 <requireAttribute>SERVER_LIST</requireAttribute>
                 <requireAttribute>CLIENT_LIST</requireAttribute>
                 <requireAttribute>PARTS_LIST</requireAttribute>
             </include>
-            <include cluster="Binding" client="true" server="false" clientLocked="false" serverLocked="true">
-                <requireAttribute>BINDING</requireAttribute>
-            </include>
             <include cluster="OTA Software Update Provider" client="false" server="true" clientLocked="true" serverLocked="true"></include>
             <include cluster="OTA Software Update Requestor" client="true" server="false" clientLocked="true" serverLocked="true"></include>
```
[the remark this answers is not in the document]
Comment (jadhavrohit924) [reply src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:154]: In specs it's mandatory.

[the remark this answers is not in the document]
Comment (jadhavrohit924) [reply src/app/zap-templates/zcl/data-model/chip/matter-devices.xml]: Okay, updated

**Review by Damian-Nordic — APPROVED** [review]

File: src/app/zap-templates/zcl/data-model/chip/matter-devices.xml [hunk 2]
```diff
@@ -144,24 +144,14 @@ limitations under the License.
...
                 <requireAttribute>SERVER_LIST</requireAttribute>
                 <requireAttribute>CLIENT_LIST</requireAttribute>
                 <requireAttribute>PARTS_LIST</requireAttribute>
             </include>
-            <include cluster="Binding" client="true" server="false" clientLocked="false" serverLocked="true">
-                <requireAttribute>BINDING</requireAttribute>
-            </include>
             <include cluster="OTA Software Update Provider" client="false" server="true" clientLocked="true" serverLocked="true"></include>
-            <include cluster="OTA Software Update Requestor" client="true" server="false" clientLocked="true" serverLocked="true"></include>
+            <include cluster="OTA Software Update Requestor" client="false" server="false" clientLocked="false" serverLocked="true"></include>
```
Comment (tcarmelveilleux) [inline src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:154]: This is client true for OTA provider

Comment (tcarmelveilleux) [reply src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:154]: Unless making client True means it's mandatory?

Comment (dhrishi) [reply src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:154]: client=true and clientLocked=true will make it mandatory. As OTA Software Update Requestor client is optional for an OTA provider, this should be client=false and clientLocked=false (default disabled, but can be enabled)

**Review by dhrishi — APPROVED** [review]

File: src/app/zap-templates/zcl/data-model/chip/matter-devices.xml (hunk 2 above)
Comment (dhrishi) [reply src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:154]: Resolving the conversation

**Pull request merged**